etc/scdaq/scdaq.conf

```diff
@@ -47,10 +49,10 @@ enable_stream_processor:yes
 # Note: When changing the processing type, change also "output_filename_prefix"
 # in the file output section.
 #
-processor_type:PASS_THROUGH
+processor_type:GMT
 
 # Enable software zero-supression
-doZS:no
+doZS:yes
```

Comment:
We expect that ZS works again after you added support for the header and trailer, right?

Reply:
yes exactly